webview-ui/src/index.css

```diff
@@ -1,4 +1,20 @@
 @import "@patternfly/react-core/dist/styles/base.css";
+.pf-v6-c-masthead,
+.pf-v6-c-page,
+.pf-v6-c-page__main-container,
+.pf-v6-c-card {
+  background-color: var(--vscode-editor-background) !important;
+}
+
+:root {
+  --pf-v6-c-card--BackgroundColor: var(--vscode-editor-background) !important;
+  --pf-t--global--background--color--primary--default: var(--vscode-editor-background) !important;
+  --pf-v6-c-content--Color: var(--vscode-editor-foreground) !important;
+  --pf-v6-c-page--BackgroundColor: var(--vscode-editor-background) !important;
+  --pf-v6-c-page__main-container--BackgroundColor: var(--vscode-editor-background) !important;
+  --pf-t--global--text--color--link--100: #3794ff; /* VS Code textLink.foreground */
+  --pf-t--global--color--status--danger--100: #f48771; /* VS Code errorForeground */
+}
```

Comment:
Ah ha, that's what I expected to see more of...overrides of the PF color/font variables.